test/stability/testDivide/testDivide.go

```diff
@@ -235,13 +246,15 @@ func executeTest(configFile, target string, testScenario uint, testType uint) er
 func setParameters(testScenario uint) (err error) {
 	eps = 3
 	passedLimit = 85
+	shift = 0
```

Comment:
too many magics, replace with names. 

By the way, logic of check packets is not obvious now, needs comments here and in place where flows are checked.

Reply:
You want to replace shift = 2 with names? Maybe with shift = two?

Reply:
shiftForSplit, shiftDefault and so on.

Reply:
And how to initialize them? "shiftForSplit" = 2 is magic number isn't it?

Reply:
yes, but when you meet 2 somewhere you understand what it is. And lower risks to forget during future changes.